cranelift/codegen/src/isa/x64/inst/mod.rs

```diff
@@ -502,7 +502,7 @@ pub(crate) fn low32_will_sign_extend_to_64(x: u64) -> bool {
 }
 
 impl Inst {
-    fn isa_requirement(&self) -> Option<InstructionSet> {
+    fn isa_requirement(&self) -> SmallVec<[InstructionSet; 1]> {
```

Comment:
Perhaps we could rename this to something like `present_in_isas()`? The `OR` semantics weren't immediately clear to me from the name until I read the code around the callsite above; otherwise I would have expected `AND`.

Reply:
Or how about `present_in_any_isa()`? (or `available_in_any_isa()`?) The "any" in the name makes the `OR` semantics crystal clear in my opinion.

Also nit: considering a `SmallVec<[T; 1]>` will cause a heap allocation for the second element, and we only have the cases for 1 and 2 elements, could this return `SmallVec<[T; 2]>` (or 4; an initial `Vec` push will reserve capacity for 4 elements)?

Reply:
Went with `available_in_any_isa`, added some documentation to that function, and updated to use `SmallVec<[T; 2]>`.